conan/internal/api/new/barebones.py

```diff
@@ -0,0 +1,11 @@
+_conanfile = '''\
+from conan import ConanFile
+
+class BarebonesConanfile(ConanFile):
+    name = "barebones"
+    version = "0.0"
+    description = "A bare-bones recipe"
```

Comment:
I like the idea of a barebones recipe that does nothing, but maybe we can use it as a empty skeleton that guides users to some simple use cases? Like adding a ``requirements()`` method with a commented out ``self.requires()``, a ``build()``, ``package()`` methods, with commented out examples and explanations?

Does this make sense? probably not worth the effort?

Maybe it is easy to allow customizations? for example, adding ``-d requires=xxx`` and let the template add requires, allow custom (but not mandatory) pkg name and version, etc.

Reply:
Sounds great, I'll get on it